packages/react-avatar/src/stories/AvatarGroupDefault.stories.tsx

```diff
@@ -1,4 +1,9 @@
 import * as React from 'react';
-import { AvatarGroup, AvatarGroupProps } from '../index';
+import { Avatar, AvatarGroup, AvatarGroupProps } from '../index';
 
-export const Default = (props: Partial<AvatarGroupProps>) => <AvatarGroup {...props} />;
+export const Default = (props: Partial<AvatarGroupProps>) => (
+  <AvatarGroup maxAvatars={1} {...props}>
+    <Avatar />
+    <Avatar />
+  </AvatarGroup>
+);
```

Comment:
Since this is the default story you should probably remove the use of `maxAvatars` and use 6 or 7 `Avatars` to get the overflow.

```suggestion
export const Default = (props: Partial<AvatarGroupProps>) => (
  <AvatarGroup {...props}>
    <Avatar />
    <Avatar />
    <Avatar />
    <Avatar />
    <Avatar />
    <Avatar />
  </AvatarGroup>
);
```